common/djangoapps/course_category/models.py

```diff
@@ -83,6 +83,25 @@ def save_reindex_course_category(sender, instance, pk_set, action, **kwargs):
         instance.pre_clear_course_keys.update(str(x.id) for x in instance.courses.all())
 
     if action in ['post_add', 'post_remove']:
-        courses_set.update(str(x.id) for x in instance.courses.all())
+        courses_set.update(str(c.id) for c in instance.courses.all())
         courses_set.update(getattr(instance, 'pre_clear_course_keys', set()))
         task_reindex_courses.delay(instance.id, list(courses_set))
+
+
+@receiver(post_init, sender=CourseCategory)
+def remember_previous_name(sender, instance, **kwargs):
+    '''
+    Saves category name to check if it was changed.
+    '''
+    instance.previous_name = instance.name
+
+
+@receiver(post_save, sender=CourseCategory)
+def category_name_change_reindex_course_category(sender, instance, created, **kwargs):
+    '''
+    We need to reindex category courses (in case of category name was changed but not category courses)
```

Comment:
Change the docstring to be compatible with pep8.
```
"""
Title.

Description.
"""
```

Reply:
Fixed.

Reply:
Docstring can't be greater than 80 chars.

Reply:
See suggestion below.